tools/rabbitmq-consumer/rabbitmq-consumer.cabal

```diff
@@ -0,0 +1,84 @@
+cabal-version: 3.0
+name:          rabbitmq-consumer
+version:       1.0.0
+synopsis:      CLI tool to consume messages from a RabbitMQ queue
+category:      Network
+author:        Wire Swiss GmbH
+maintainer:    Wire Swiss GmbH <backend@wire.com>
+copyright:     (c) 2023 Wire Swiss GmbH
+license:       AGPL-3.0-only
+build-type:    Simple
+
+executable rabbitmq-consumer
+  main-is:        Main.hs
+  build-depends:
+    , base
+    , rabbitmq-consumer
+
+  hs-source-dirs: app
+
+library
+  hs-source-dirs:     src
+  exposed-modules:    RabbitMQConsumer.Lib
+  default-language:   GHC2021
+  ghc-options:
+    -Wall -Wpartial-fields -fwarn-tabs
+    -optP-Wno-nonportable-include-path
+
+  build-depends:
+    , aeson
+    , aeson-pretty
+    , amqp
+    , base
+    , bytestring
+    , imports
+    , network
+    , optparse-applicative
+    , text
+    , types-common
+    , wire-api
+    , wire-api-federation
+
+  default-extensions:
```

Comment:
At some point we should go around cleaning up these extensions since a lot of them are on by default with GHC2021, right?